src/staging/staging_main.py

```diff
@@ -209,15 +209,6 @@ def run_staging(  # noqa: C901
 
     StagingMainLogger.info("Backdata File Loaded Successfully...")
 
-    # Loading ITL1 detailed mapper
```

Comment:
Please check that itl1_detailed_mapper is removed from the directory in DAP and from the res_dev_runner config.

Reply:
I'll do it in DAP when we do the new release.